src/stan/callbacks/unique_stream_writer.hpp

```diff
@@ -66,17 +67,34 @@ class unique_stream_writer final : public writer {
    *
    * @param[in] state Values in a std::vector
    */
-  void operator()(const std::vector<double>& state) { write_vector(state); }
+  void operator()(const std::vector<double>& state) {
+    if (output_ == nullptr)
+      return;
+    write_vector(state);
+  }
 
-  /**
-   * Writes the comment_prefix to the stream followed by a newline.
-   */
-  void operator()() {
+  void operator()(const std::tuple<Eigen::VectorXd, Eigen::VectorXd>& state) {
```

Comment:
What is the purpose of this overload?

Reply:
updated unique_stream_writer.hpp - the above was present in an old version of pathfinder, checked in by mistake.